Tests/Unit/FrontEnd/DefaultControllerTest.php

```diff
@@ -1210,10 +1210,6 @@ public function testSingleViewForSeminarWithoutImageDoesNotDisplayImage()
         );
         $this->fixture->piVars['showUid'] = $this->seminarUid;
 
-        self::assertNotContains(
-            'style="background-image:',
-            $this->fixture->main('', [])
-        );
```

Comment:
Please don't remove these tests. Instead, please replace them to test for the new content. (Otherwise, the tests won't test that the image works anymore, i.e., they don't ensure that this works fine and won't break in the future.)